src/features/views/components/ViewSurveySubmissionPreview/index.tsx

```diff
@@ -79,7 +79,15 @@ const ViewSurveySubmissionPreview: FC<ViewSurveySubmissionPreviewProps> = ({
                 >
                   <ZUIRelativeTime datetime={mostRecent.submitted} forcePast />
                 </Typography>
-                <Box marginBottom={1}>{mostRecent.matchingContent}</Box>
+                <Box marginBottom={1}>
+                  {mostRecent.matchingContent ? (
+                    mostRecent.matchingContent
+                  ) : (
+                    <Typography color="secondary" sx={{ fontStyle: 'italic' }}>
+                      <FormattedMessage id="misc.views.surveyPreview.matches.notSelected" />
+                    </Typography>
+                  )}
+                </Box>
```

Comment:
This should not be handled by `ViewSurveySubmissionPreview`. Just pass the `<Typography>…` as the `matchingContent` of the first submission.